vizro-core/tests/unit/vizro/models/_navigation/test_accordion.py

```diff
@@ -41,46 +41,29 @@ def test_field_invalid_pages_empty_list(self):
 class TestAccordionBuild:
     """Tests accordion build method."""
 
-    def test_accordion_build_default(self, accordion_from_page_as_list):
-        accordion_div = Accordion(id="accordion_list")
-        component = accordion_div.build()
-
-        result = json.loads(json.dumps(component, cls=plotly.utils.PlotlyJSONEncoder))
+    @pytest.mark.parametrize("pages", [["Page 1", "Page 2"], None])
+    def test_accordion_build_default(self, pages, accordion_from_page_as_list):
+        accordion = Accordion(pages=pages, id="accordion_list").build()
+        result = json.loads(json.dumps(accordion, cls=plotly.utils.PlotlyJSONEncoder))
         expected = json.loads(json.dumps(accordion_from_page_as_list, cls=plotly.utils.PlotlyJSONEncoder))
-
         assert result == expected
 
     def test_accordion_build_pages_as_list(self, pages_as_list, accordion_from_page_as_list):
-        accordion_div = Accordion(pages=pages_as_list, id="accordion_list")
-        component = accordion_div.build()
-
-        result = json.loads(json.dumps(component, cls=plotly.utils.PlotlyJSONEncoder))
+        accordion = Accordion(pages=pages_as_list, id="accordion_list").build()
+        result = json.loads(json.dumps(accordion, cls=plotly.utils.PlotlyJSONEncoder))
         expected = json.loads(json.dumps(accordion_from_page_as_list, cls=plotly.utils.PlotlyJSONEncoder))
-
         assert result == expected
 
     def test_accordion_build_pages_as_dict(self, pages_as_dict, accordion_from_pages_as_dict):
-        accordion_div = Accordion(pages=pages_as_dict, id="accordion_dict")
-        component = accordion_div.build()
-
-        result = json.loads(json.dumps(component, cls=plotly.utils.PlotlyJSONEncoder))
+        accordion = Accordion(pages=pages_as_dict, id="accordion_dict").build()
+        result = json.loads(json.dumps(accordion, cls=plotly.utils.PlotlyJSONEncoder))
         expected = json.loads(json.dumps(accordion_from_pages_as_dict, cls=plotly.utils.PlotlyJSONEncoder))
-
         assert result == expected
 
+    def test_accordion_build_single_page_accordion(self, single_accordion_single_page):
```

Comment:
Are we using `single_accordion_single_page` fixture in this test?

Reply:
Ah, you are right! Good catch, I added that before i just created the accordion in the test.